config/purposes/scrna_core_cdna_prep.wip.yml

```diff
@@ -26,22 +26,8 @@ LRC PBMC Defrost PBS:
   :asset_type: plate
   :label_template: plate_quad_qc # creates QC4 barcodes
   :file_links:
-    - name: 'Download Cellaca Input QC1'
-      id: cellaca_input_file
-      params:
-        page: 0
-    - name: 'Download Cellaca Input QC2'
-      id: cellaca_input_file
-      params:
-        page: 1
-    - name: 'Download Cellaca Input QC3'
-      id: cellaca_input_file
-      params:
-        page: 2
-    - name: 'Download Cellaca Input QC4'
-      id: cellaca_input_file
-      params:
-        page: 3
+    - name: 'Download Cellaca CSV'
```

Comment:
Check the name for this with the users? My understanding was this is not the Cellaca file but the driver file to create the plate that will be loaded onto the Cellaca, and that they will create the Cellaca file outside the LIMS. Maybe the name should reflect that distinction?
e.g Download Hamilton CSV File to Create Cellaca QC Plate

Reply:
Renamed to Download Cellaca Count CSV